GitUI/GitUIExtensions.cs

```diff
@@ -144,6 +138,20 @@ void OpenWithDiffTool()
                 return file.IsSubmodule
                     ? LocalizationHelpers.ProcessSubmodulePatch(fileViewer.Module, file.Name, patch)
                     : patch?.Text;
+
+                static Patch GetItemPatch(
```

Comment:
❔ Is this used anywhere?

Reply:
GetItemPatch is used in GetSelectedPatch (why it was made into a local function)